dev-scripts/minishift_deploy.sh

```diff
@@ -71,7 +71,6 @@ oc create configmap che \
       --from-literal=che-server-java-opts="-XX:+UseG1GC -XX:+UseStringDeduplication -XX:MinHeapFreeRatio=20 -XX:MaxHeapFreeRatio=40 -XX:MaxRAM=700m -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap" \
       --from-literal=che-workspaces-java-opts="-XX:+UseG1GC -XX:+UseStringDeduplication -XX:MinHeapFreeRatio=20 -XX:MaxHeapFreeRatio=40 -XX:MaxRAM=1300m -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap" \
       --from-literal=che-openshift-secure-routes="false" \
-      --from-literal=che-secure-external-urls="false" \
```

Comment:
Ah it's because the corresponding environment value is still there in the `fabric8-online` deployment yam (for compatibility of course). So let's keep this line for the moment even if it's not used anymore.